srcs/image/rendition.c

```diff
@@ -0,0 +1,66 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   rendition.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/09/28 16:32:52 by elouisia          #+#    #+#             */
+/*   Updated: 2022/10/05 12:06:33 by elouisia         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+void	ft_render_background(t_cub_data *data, int colour)
+{
+	int	x;
+	int	y;
+
+	y = 0;
+	while (y < HEIGHT)
+	{
+		x = 0;
+		while (x < WIDTH)
+		{
+			ft_put_pix_img(&data->img, x, y, colour);
+			x++;
+		}
+		y++;
+	}
+}
+
+void	ft_render_player(t_cub_data *data, int colour)
+{
+	int	i;
+	int	j;
+
+	i = data->player.y;
+	while (i < data->player.y + 10)
+	{
+		j = data->player.x;
+		while (j < data->player.x + 10)
+		{
+			ft_put_pix_img(&data->img, j, i, colour);
+			j++;
+		}
+		i++;
+	}
+}
+
+void	ft_init_player(t_cub_data *data)
+{
+	data->player.x = WIDTH / 2;
+	data->player.y = HEIGHT / 2;
+}
+
+int	ft_render_img(t_cub_data *data)
+{
+	ft_render_background(data, 0xA9A9A9);
+	ft_render_player(data, 0x8E1600);
+	// ft_set_ray_data(data);
```

Comment:
This comment should not be here merged on master.

We have no hooks yet, but with hooks any file not at norm would be rejected.

Reply:
Done !